docs/non_containerized_NSFS.md

```diff
@@ -57,19 +57,51 @@ rpm -i noobaa-core-5.14.0-1.el8.x86_64.rpm
 
 After installing NooBaa RPM, it's expected to have noobaa-core source code under /usr/local/noobaa-core and an nsfs systemd example script under /etc/systemd/system/.
 
+## Create configuration files -
+**IMPORTANT NOTE** - It's not recommended to create the config_root under /tmp/ since the contents of /tmp/ can be deleted occasionaly, In the following instruction we use it just as an example. 
+
+**1. Create buckets and accounts directories -**
+```sh
+mkdir -p /tmp/noobaa_config_root/
+mkdir -p /tmp/noobaa_config_root/buckets/
+mkdir -p /tmp/noobaa_config_root/accounts/
+```
+
+**2. Create accounts and exported buckets configuration files -**
+
+Find instruction at - https://github.com/noobaa/noobaa-core/blob/master/docs/nsfs-standalone.md. <br />
+**Note** - All required paths on the configuration files (bucket - path, account - new_buckets_path) must be absolute paths.
+
+
+## Create FS -
+If it's not already existing, create the fs root path in which buckets (directories) and objects (files) will be created.
+
+```sh
+mkdir -p /tmp/fs1/
+```
+
+
 ## Run the nsfs service - 
-This systemd example script runs noobaa non containerized (currently) single mode, while /tmp/test/ is the fs root path directory.
+The systemd script runs noobaa non containerized, and requires config_root in order to find the location of the system/accounts/buckets configuration file.
+
+```sh
+systemctl start nsfs --config_root=/tmp/noobaa_config_root
```

Comment:
@romayalon do we need to mention config_root in systemctl?

Reply:
we need to remove it in - https://github.com/noobaa/noobaa-core/pull/7509
i'll do that, thanks!